scripts/common.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+function format_json() {
+    local raw_data="$1"
+    local closing_curly_bracket="}"
+
+    # Replace parts to form valid JSON objects
```

Comment:
You should put an example of the raw_data in a comment so we can follow the operations.

Reply:
yes this was just extracted from `queue_runner.sh` to be reused in `extract_ids.sh`